src/api/digitalMedia/GetDigitalMedia.ts

```diff
@@ -0,0 +1,50 @@
+/* Import Dependencies */
+import axios from 'axios';
+
+/* Import Types */
+import { DigitalMedia } from 'app/types/DigitalMedia';
+import { JSONResult } from 'app/Types';
+
+/* Import Exceptions */
+import { NotFoundException } from 'app/Exceptions';
+
+
+/**
+ * Function for fetching a digital media item
+ * @param handle The identifier of the digital media item
+ * @param version The version of the digital media item
+ * @returns Object of Digital Media
+ */
+const GetDigitalMedia = async({ handle, version } : { handle: string, version?: number }) => {
+    let digitalMedia: DigitalMedia | undefined;
+
+    if (handle) {
+        let endPoint: string;
+
+        if (version) {
+            endPoint = `digital-media${handle}/${version}`;
```

Comment:
missing a slash between digital media and $handle ->  digital-media/${handle}/${version}